docs/guide/field_type_reference.md

```diff
@@ -393,11 +473,11 @@ $settings = array(
 
 ### Template rendering
 
-The template called by [the `ez_render_field()` Twig function](content_rendering.md/#ez_render_field) while rendering a Date Field has access to the following parameters:
+The template called by [the `ez_render_field()` Twig function](../guide/content_rendering.md/#ez_render_field) while rendering a Date Field has access to the following parameters:
```

Comment:
This time th change is not necessary, because we are inside /guide/ already...
But the link is broken, I think, because it should be `(content_rendering.md#ez_render_field)`, without the backslash